### PR TITLE
add collection listing

### PR DESCRIPTION
**Pull request by robjloranger**
adds subcommand colls
lists collections for authenticated user
supports -url flag for listing collection URLs

### CODE REVIEW

**Comment by thebaer** [comment]
Have only read the code, but it's looking good so far. Only feedback is I'd say we make the subcommand "blogs" instead of "colls", since "collections" is only used internally.

**Comment by robjloranger** [comment]
updated @thebaer

File: api/api.go [hunk 1]
```diff
@@ -140,6 +140,33 @@ func DoPost(c *cli.Context, post []byte, font string, encrypt, tor, code bool) (
 	return p, nil
 }
 
+// DoFetchCollections retrieves a list of the currently logged in users
+// collections.
+func DoFetchCollections(c *cli.Context) ([]RemoteColl, error) {
+	cl, err := NewClient(c, true)
+	if err != nil {
+		//
```
Comment (thebaer) [inline api/api.go]: We should return the `err` here

File: api/api.go [hunk 2]
```diff
@@ -140,6 +140,33 @@ func DoPost(c *cli.Context, post []byte, font string, encrypt, tor, code bool) (
...
+	}
+
+	colls, err := cl.GetUserCollections()
+	if err != nil {
+		//
```
Comment (thebaer) [inline api/api.go]: (and here)

**Comment by robjloranger** [comment]
all set here @thebaer

**Comment by thebaer** [comment]
Looks great. Thanks!

**Pull request merged**